conans/client/generators/json_generator.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from conans.model import Generator
+
+
+def serialize_cpp_info(cpp_info):
+    keys = [
+        "version",
+        "description",
+        "rootpath",
+        "sysroot",
+        "include_paths", "lib_paths", "bin_paths", "build_paths", "res_paths",
```

Comment:
Better stick to same names as in _conaninfo.txt_: includedirs, libdirs, bindirs...

Reply:
Agreed.